### PR TITLE
Set up factory gem for testing env

### PR DESCRIPTION
**Pull request by anatoliikryvishyn**

### CODE REVIEW

**Comment by anatoliikryvishyn** [comment]
@serhiir , please [test](https://tada-interlink.herokuapp.com/ ) and review.

File: test/view_objects/list_view_test.rb [hunk 1]
```diff
@@ -1,13 +1,17 @@
...
-    assert_equal [items(:sleep), items(:shower),items(:dinner)], ListView.new(lists(:home)).uncompleted_items
+    assert_equal [build(:sleep), build(:shower),build(:dinner)], ListView.new(build(:home)).uncompleted_items
   end
 
   test "order of completed items should be from last modified to earlier modified" do
-    assert_equal [items(:university),items(:lunch)], ListView.new(lists(:office)).completed_items
+    assert_equal [build(:university),build(:lunch)], ListView.new(build(:office)).completed_items
   end
 
   test "button text shoud be correct when list is empty" do
```
Comment (serhiir) [inline test/view_objects/list_view_test.rb]: @anatoliikryvishyn What about `button text shoud be correct when list is empty` and `button text shoud be correct when list is not empty` tests? Should we steal use `lists(:university)` and `lists(:office)` fixtures for them?

**Comment by serhiir** [comment]
@anatoliikryvishyn, As far as I remember [old fixutes](https://github.com/anatoliikryvishyn/TaDa/tree/6-setup-factory-girl/test/fixtures) (both) were kept for rake task, right?

**Comment by serhiir** [comment]
@anatoliikryvishyn Done.

**Comment by anatoliikryvishyn** [comment]
>@anatoliikryvishyn, As far as I remember old fixutes (both) were kept for rake task, right?

@serhiir, yes their usage is just to fill test db.

**Comment by anatoliikryvishyn** [comment]
@serhiir, updated.

**Pull request merged**